istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -350,6 +352,10 @@ template: |
   {{- end }}
     {{  end -}}
     volumeMounts:
+    {{- if not .Values.global.signCertAtKubernetesCa }}
+    - mountPath: /etc/citadel-ca-cert
```

Comment:
nit: maybe make this /etc/istio/citadel-ca-cert? so we don't have so much files in random places

Reply:
Done.